packages/vite/src/node/plugins/workerImportMetaUrl.ts

```diff
@@ -20,23 +20,23 @@ type WorkerType = 'classic' | 'module' | 'ignore'
 const WORKER_FILE_ID = 'worker_url_file'
 
 function getWorkerType(code: string, i: number): WorkerType {
-  const commaIndex = findIndex(code, i, ',')
+  const clearCode = code
+    .slice(i)
+    .replace(singlelineCommentsRE, '')
+    .replace(multilineCommentsRE, '')
+
```

Comment:
We can directly pass the `noCommentCode` as clean code
https://github.com/vitejs/vite/blob/87c9e3d08e10b31f52245ed7efd8fe8708d8bc4b/packages/vite/src/node/plugins/workerImportMetaUrl.ts#L38-L40